TunnelVisionLabs.ReferenceAssemblyAnnotator/AnnotatorBuildTask.cs

```diff
@@ -104,7 +104,14 @@ public override bool Execute()
             string targetDocumentation = Path.ChangeExtension(outputAssembly, ".xml");
             if (File.Exists(sourceDocumentation))
```

Comment:
Why not this?

```suggestion
            if (File.Exists(sourceDocumentation) && !File.Exists(targetDocumentation))
```

Reply:
We can sure add that, but we can't rely on File.Exists because we might be racing with a concurrent build process.

Reply:
It would be fine to have both for now. The exception handling could be removed when the machine caching strategy is implemented.